packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -442,30 +441,29 @@ export class ModifierOpcode extends Opcode {
 
 export class UpdateModifierOpcode extends UpdatingOpcode {
   public type = "update-modifier";
-
   private element: Element;
   private dynamicScope: DynamicScope;
   private args: EvaluatedArgs;
   private manager: ModifierManager<Opaque>;
   private modifier: Opaque;
   private lastUpdated: Revision;
 
-  constructor({ manager, modifier, element, dynamicScope, args }: { manager: ModifierManager<Opaque>, modifier: Opaque, element: Element, dynamicScope: DynamicScope, args: EvaluatedArgs }) {
+  constructor({ manager, modifier, element }: { manager: ModifierManager<Opaque>, modifier: Opaque, element: Element }) {
```

Comment:
Can you change this to positional args while you are at it? We have been doing that whenever we touch existing code.